### PR TITLE
fix click events on the nav's in Firefox

### PR DESCRIPTION
**Pull request by urkle**
- the global event object is non-standard (and not implemented in Firefox)
- use the first parameter to the event callback instead (as this is the standard)

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -9,10 +9,10 @@
 //= require clipboard
 //= require_tree .
 
-function handleClick(event, nav, removeNavExpandedClass, addNavExpandedClass) {
+function handleClick(e, nav, removeNavExpandedClass, addNavExpandedClass) {
```
Comment (jvanbaarsen) [inline app/assets/javascripts/application.js]: What is the reason for this change? I would say that `event` is more explicit (and thus more clear).

Comment (urkle) [reply app/assets/javascripts/application.js]: everywhere else is using "e" so I was going for consistency.   I can revert this if desired.

Comment (urkle) [reply app/assets/javascripts/application.js]: @jvanbaarsen did a force push removing the changes from this file.

**Comment by sonalkr132** [comment]
Thanks @urkle
